src/components/Filters/StatefulFilters.tsx

```diff
@@ -234,7 +234,10 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
   };
 
   removeFilter = (id: string, value: string) => {
+    console.log('Remove ' + id + ' _ ' + value);
+    console.log(this.state.activeFilters.filters);
     const updated = this.state.activeFilters.filters.filter(x => x.id !== id || x.value !== value);
+    console.log(updated);
```

Comment:
Need to remove.

Reply:
done @jshaughn I forgot remove it for this PR sorry and thanks